### PR TITLE
Bug Fix: Activity Log Tasks Modal displays Tasks from day before lower bound date

### PR DESCRIPTION
**Pull request by davidnmora**
## Bug:
Christina emailed me pointing out that the Activity Log returns Tasks (in the Task modal) from __before__ the lower bound date (eg Aug 14th-Aug 14th query includes results from Aug 13th). This is because we were querying for Tasks Aug 14th-Aug 14th in local time, which returns earlier tasks when interpreted as UTC (meaning we could possibly leave out Tasks created __late__ Aug 14th, too).

## Fix:
I simply refactored and re-used existing code to fix it.

Note that I changed our Task FilterSet to _DateTimeFilter_ from DateFilter to prevent this issue from happening in the future. Though, for that to work, `created_at` needs to be specified in ISO (which is in UTC and has timezone), so to be honest I'm unsure if this fixes the problem... would love thoughts on this.

### CODE REVIEW

**Review by ishitap — DISMISSED** [review]

File: ics/v11/queries/tasks.py [hunk 1]
```diff
@@ -3,10 +3,21 @@
...
 import datetime
+import pytz
+
+
+def filter_by_created_at_range(query_params, queryset):
+	start = query_params.get('start', None)
+	end = query_params.get('end', None)
+	if start is not None and end is not None:
+		dt = datetime.datetime
+		start_date = pytz.utc.localize(dt.strptime(start, constants.DATE_FORMAT))
```
Comment (mayanb) [inline ics/v11/queries/tasks.py:14]: we should convert the start date into the start of that day for the start date in the team's timezone and the end date into the start of that day in that team's timezone - we're storing the timezones in the Team model

**Review by mayanb — DISMISSED** [review]
use team's timezones

File: ics/v11/queries/tasks.py (hunk 1 above)
Comment (davidnmora) [reply ics/v11/queries/tasks.py:14]: I think we have already done this on the frontend:

Comment (davidnmora) [reply ics/v11/queries/tasks.py:14]: This is how we format the date we send over, which gives the start and end of the day in local format:
```javascript

export function dateToUTCString(dateString, addOne) {
	var m = moment(dateString + " 00:00:00")

	if (addOne) {
		m.add(24, "hours")
	}

	return toUTCString(m)
}

export function toUTCString(momentDate) {
	return momentDate.utc().format('YYYY-MM-DD-HH-mm-ss-SSSSSS')
}

**Review by mayanb — APPROVED** [review]

**Pull request merged**